Dockerfile

```diff
@@ -0,0 +1,26 @@
+# building frontend
+FROM node:13.1 as build-deps
+COPY web/package.json web/yarn.lock ./
```

Comment:
Why do you first copy only this files then the whole folder?

Reply:
fixed